pkg/volume/csi/csi_plugin.go

```diff
@@ -150,7 +150,6 @@ func (h *RegistrationHandler) RegisterPlugin(pluginName string, endpoint string,
 		klog.Error(log("registrationHandler.RegisterPlugin failed at CSI.NodeGetInfo: %v", err))
```

Comment:
Can we also remove this log statement?  I don't think we need to log an error that's going to get logged later in the stack.

Reply:
Done.

Reply:
sorry I meant remove the log of registerplugin err that we are returning

Reply:
Done. But note that this practice (log & return the err) is widely used in this package. I might clean this out in another PR if it's desirable.

Reply:
Thanks! Yes, I think we should clean up the rest of the package